category/application/fake/category_service.go

```diff
@@ -0,0 +1,75 @@
+package fake
```

Comment:
Usually our fake implementations life under infrastructure/fake/ we should move this as well to be more inline with the other modules.